ruby.rb

```diff
@@ -5,41 +5,73 @@
 
 myArray = [1, 2, 6, 9, 3, 21]
 
+p myArray.map { |number| number * 3}
 
+myArray.each do |arr|
+    arr * 3
+    p myArray
+end
 
 # 2. Create a method that takes in a sentence and returns a new sentence with the first letter of each word capitalized. Expected output = "Hello There, How Are You?"
 
 sentence = "hello there, how are you?"
 
+# I am not sure if I am using the capitalize correctly.
+def cap_it
+  words = "hello there, how are you?"
+  p words.split(' ').capitalize.join(' ')
+end
+
+def cap_it
+    sentence = "hello there, how are you?"
+  p sentence.capitalize
+end
+p cap_it
+# THIS ONLY RETURNS THE FIRST WORD CAPITALIZED
 
 
 # 3. Create a method that takes in a string and returns a new string with all the vowels removed. Expected output = " hv n vwls"
 
-no_vowels = "I have no vowels"
 
+def remove_vowels
+    no_vowels = "I have no vowels"
+    p no_vowels.remove('aeiou')
+end
+p remove_vowels
 
```

Comment:
I think you are looking for the delete method here. And don't forget about uppercase vowels!